### PR TITLE
Update some vtctl/vtctlclient command help

### PR DESCRIPTION
**Pull request by aquarapid**

### CODE REVIEW

File: go/vt/vtctl/vtctl.go [hunk 1]
```diff
@@ -2351,8 +2351,8 @@ func commandVerticalSplitClone(ctx context.Context, wr *wrangler.Wrangler, subFl
 }
 
 func commandVDiff(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.FlagSet, args []string) error {
-	sourceCell := subFlags.String("source_cell", "", "The source cell to compare from")
-	targetCell := subFlags.String("target_cell", "", "The target cell to compare with")
+	sourceCell := subFlags.String("source_cell", "", "The source cell to compare from; default is all cells")
+	targetCell := subFlags.String("target_cell", "", "The target cell to compare with; default is all cells")
```
Comment (deepthi) [inline go/vt/vtctl/vtctl.go:2355]: Isn't all cells misleading? we would choose one out of all available.

Comment (aquarapid) [reply go/vt/vtctl/vtctl.go:2355]: I wasn't trying to comment on the effect of the default setting;  just that the default is all cell cells. Will adjust.  The docs have the same issue, BTW.

**Comment by deepthi** [comment]
Now you have a merge conflict..

File: go/vt/vtctl/vtctl.go [hunk 2]
```diff
@@ -334,7 +334,7 @@ var commands = []commandGroup{
 				"<from_keyspace> <to_keyspace> <tables>",
 				"Start the VerticalSplitClone process to perform vertical resharding. Example: SplitClone from_ks to_ks 'a,/b.*/'"},
 			{"VDiff", commandVDiff,
-				"[-source_cell=<cell>] [-target_cell=<cell>] [-tablet_types=replica] [-filtered_replication_wait_time=30s] <keyspace.workflow>",
+				"[-source_cell=<cell>] [-target_cell=<cell>] [-tablet_types=master,replica,rdonly] [-filtered_replication_wait_time=30s] <keyspace.workflow>",
```
Comment (deepthi) [inline go/vt/vtctl/vtctl.go:337]:
```suggestion
				"[-source_cell=<cell>] [-target_cell=<cell>] [-tablet_types=primary,replica,rdonly] [-filtered_replication_wait_time=30s] <keyspace.workflow>",
```

**Review by deepthi — APPROVED** [review]

**Pull request merged**